packages/eslint-plugin/src/tests/atom-postfix-rule.test.ts

```diff
@@ -0,0 +1,124 @@
+import { RuleTester } from 'eslint'
+import { atomPostfixRule } from '../rules/atom-postifx-rule'
+
+const tester = new RuleTester({
+  parserOptions: {
+    ecmaVersion: 2022,
+    sourceType: 'module',
+  },
+  settings: {
+    atomPostfix: 'Atom'
+  }
+})
+
+tester.run('reatom/atom-postfix-rule', atomPostfixRule, {
+  valid: [
+    {
+      code: `
+            import { atom } from '@reatom/framework'
+            const countAtom = atom(0, "countAtom");
+            `,
+    },
+    {
+      code: `const countAtom = atom(0);`,
+    },
+    {
+      code: 'const countAtom = atom(0, "count");',
```

Comment:
shouldn't it have `countAtom` here or it's not reatom exported atom and it should not be considered?

Reply:
it's variable postfix checker, not name in the call. 
This test is not suitable)
I'll delete one test